calico.go

```diff
@@ -191,7 +192,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 			// Parse endpoint labels passed in by Mesos, and store in a map.
 			labels := map[string]string{}
 			for _, label := range conf.Args.Mesos.NetworkInfo.Labels.Labels {
-				labels[label.Key] = label.Value
+				k := strings.Replace(label.Key, "/", ".", -1)
```

Comment:
Probably worth a quick comment along the lines of "Slashes are allowed in Mesos labels but not in Calico ones"